src/api/endpoint/api/star.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import type {$Response} from 'express';
+import type {$RequestExtend, $NextFunctionVer, IStorageHandler} from '../../../../types';
+
+export default function(storage: IStorageHandler) {
+  return (req: $RequestExtend, res: $Response, next: $NextFunctionVer): void => {
+    const name = req.params.package;
+    // Check is star or unstar
+    const isStar = Object.keys(req.body.users).indexOf(req.remote_user.name) >= 0;
+    const afterChangePackage = function(err) {
+      if (err) {
+        return next(err);
+      }
+      res.status(200);
+      next({
+        success: true,
+      });
+    };
+
+    storage.getPackage({
+      name,
+      req,
+      callback: function(err, info) {
+        if (err) {
+          return next(err);
+        }
+        let newUsers = {
+          ...info.users,
+        };
+        if (isStar) {
+          newUsers = {
+            ...newUsers,
+            [req.remote_user.name]: true,
```

Comment:
`req.remote_user.name` is used three times, I'd rather save it as variable above and reuse the variable.